launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -22,13 +22,16 @@ class ForkScaffolder(private val launcher: Launcher) {
             localInstance.coreComponentsUrl={{ localInstanceCoreComponentsUri }}
             localInstance.openMode={{ localInstanceOpenMode }}
             instance.default.type={{instanceType}}
+            instance.default.runModes={{ localInstanceRunModes }}
             instance.default.password={{instancePassword}}
             instance.local-author.enabled={{instanceAuthorEnabled}}
             instance.local-author.httpUrl={{instanceAuthorHttpUrl}}
             instance.local-author.openPath=/aem/start.html
             instance.local-publish.enabled={{instancePublishEnabled}}
             instance.local-publish.httpUrl={{instancePublishHttpUrl}}
             instance.local-publish.openPath=/crx/packmgr
+            
+            javaSupport.version=11
```

Comment:
how about putting it to the gradle.properties instead?

Reply:
on it